src/Neo/Neo.csproj

```
@@ -8,7 +8,8 @@
 
   <ItemGroup>
     <PackageReference Include="Akka" Version="1.5.36" />
-    <PackageReference Include="BouncyCastle.NetCore" Version="2.2.1" />
+    <!-- DO NOT CHANGE BouncyCastle.NetCore VERSION FROM 1.9.0, IT IS NEEDED BY NeoFS.API -->
+    <PackageReference Include="BouncyCastle.NetCore" Version="1.9.0" />
```

```suggestion
    <PackageReference Include="BouncyCastle.NetCore" Version="2.2.1" />
```
Move neo-fs in a different PR to solve this issue